components/tyk-bootstrap/templates/bootstrap-post-install.yaml

```diff
@@ -29,18 +29,19 @@ spec:
       serviceAccountName: k8s-bootstrap-role
       containers:
         - name: bootstrap-tyk-post-install
-          image: tykio/tyk-k8s-bootstrap-post:latest
+          {{/*  emergency solution so it will work with the new k8s bootstrap modifications*/}}
+          image: andreiattyk/tyk-repo:post33
```

Comment:
can't we publish officially yet?

Reply:
We can discuss releasing the k8s-bootstrap update today